src/main/java/coloringpage/Frame.java

```diff
@@ -1,4 +1,130 @@
 package coloringpage;
 
-public class Frame {
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class Frame extends JFrame {
+
+    private JPanel topPanel;
+    private JTextField pathField;
+
+    private JPanel middlePanel;
+    private final JLabel originalImageLabel = new JLabel();
+    private final JLabel finalImageLabel = new JLabel();
+
+    private JPanel bottomPanel;
+
+    ImageController controller = new ImageController(originalImageLabel, finalImageLabel);
+    private final  JFileChooser fileChooser = new JFileChooser();
+
+    public Frame() {
+        super();
+        setSize(800, 500);
+        setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+        setTitle("Coloring Page Maker");
+        setLayout(new BorderLayout());
+
+        setTopPanel();
+        setImagesPanel();
+        setBottomPanel();
+
+
+        add(topPanel, BorderLayout.PAGE_START);
+        add(middlePanel, BorderLayout.CENTER);
+        add(bottomPanel, BorderLayout.PAGE_END);
+    }
+
+    private void setTopPanel() {
+        topPanel = new JPanel();
+        JLabel imagePathLabel = new JLabel("Enter image url or Browse:");
```

Comment:
maybe lowercase b or uppercase everything else?

Reply:
If it looks awkward then I'll switch to lowercase. Thought uppercase because the button is Browse with an uppercase, but its a small thing